### PR TITLE
Callback Lesson Complete

### PR DESCRIPTION
**Pull request by dionew1**
I added as seed file, but I'm not sure if we want to keep it or not.
Just let me know your thoughts @iandouglas.

### CODE REVIEW

File: app/controllers/songs_controller.rb [hunk 1]
```diff
@@ -4,6 +4,11 @@ def index
     @songs = Song.all
   end
 
+  def show
+    binding.pry
```
Comment (iandouglas) [inline app/controllers/songs_controller.rb]: can we remove this?

**Review by iandouglas — COMMENTED** [review]
I'm okay with the seed file being there

File: app/controllers/songs_controller.rb (hunk 1 above)
Comment (dionew1) [reply app/controllers/songs_controller.rb]: Yes, definitely.

**Comment by dionew1** [comment]
Okay, tests are all passing and the slug and callback are complete. 
Pry is no longer in there either 😄

**Pull request merged**